python/lsst/ts/scheduler/stateMachine/target_producer.py

```diff
@@ -0,0 +1,90 @@
+import time
+import threading
+import logging
+
+from SALPY_scheduler import *
```

Comment:
I don't think it is a good idea to `import *`. Specially, note that you are only using a couple of the definitions here, `SAL_scheduler` and `scheduler_logevent_targetC` so you can easily import them individually.

Reply:
good point, done